app/aux_server.go

```diff
@@ -29,6 +29,7 @@ type AuxServer struct {
 	Addr     string // TCP address to listen on, ":http" if empty
 	Gatherer prometheus.Gatherer
 	Debug    bool
+	ready    bool // Readiness handler for Kubernetes
```

Comment:
This field is read from web server's goroutines and mutated from generic controller's goroutine without any mutexes. It's a data race =)